### PR TITLE
region-grower update

### PR DESCRIPTION
**Pull request by arnaudon**
Update region-grower and dependencies to latest versions.

### CODE REVIEW

**Comment by arnaudon** [comment]
Just waiting for region-grower release.

File: var/spack/repos/builtin/packages/py-morph-tool/package.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class PyMorphTool(PythonPackage):
     url      = "https://pypi.io/packages/source/m/morph-tool/morph-tool-2.4.4.tar.gz"
 
     version('develop', branch='master')
+    version('2.5', sha256='279123b59797132a2e4f8fd9df391279fce85891dec046bd1206de4c7d4f8e11')
```
Comment (arnaudon) [inline var/spack/repos/builtin/packages/py-morph-tool/package.py]:
```suggestion
    version('2.5.0', sha256='279123b59797132a2e4f8fd9df391279fce85891dec046bd1206de4c7d4f8e11')
```

File: var/spack/repos/builtin/packages/py-region-grower/package.py [hunk 2]
```diff
@@ -22,13 +22,13 @@ class PyRegionGrower(PythonPackage):
...
+    depends_on('py-tns@2.4.2:', type='run')
     depends_on('py-voxcell@2.7:', type='run')
-    depends_on('py-diameter-synthesis@0.2.3:', type='run')
+    depends_on('py-diameter-synthesis@0.2.4:', type='run')
     depends_on('py-morphio@2.7:', type='run')
-    depends_on('py-neuroc@0.2.3:', type='run')
+    depends_on('py-neuroc@0.2.6:', type='run')
     depends_on('py-neurom@2:2.999', type='run')
-    depends_on('py-morph-tool@2.4.1:', type='run')
+    depends_on('py-morph-tool@2.5:', type='run')
```
Comment (arnaudon) [inline var/spack/repos/builtin/packages/py-region-grower/package.py]:
```suggestion
    depends_on('py-morph-tool@2.5.0:', type='run')
```

**Review by matz-e — APPROVED** [review]

**Comment by arnaudon** [comment]
I'll merge if it's ok with @GianlucaFicarelli!

**Pull request merged**